File: android/src/main/java/com/getcapacitor/community/classes/events/PeerIDEvent.java
package com.getcapacitor.community.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;

public class PeerIDEvent {

    @NonNull
    String peerID;

    public PeerIDEvent(@NonNull String peerID) {
        this.peerID = peerID;
    }

    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        result.put("peerID", peerID);

        return result;
    }
}

File: ios/Sources/BitchatPlugin/Classes/Events/PeerIDEvent.swift
import Foundation
import Capacitor

@objc public class PeerIDEvent: NSObject {
    let peerID: String

    init(_ peerID: String) {
        self.peerID = peerID
    }

    public func toJSObject() -> JSObject {
        var result = JSObject()

        result["peerID"] = peerID

        return result
    }
}
